src/services/exit-logs/verifier.ts

```diff
@@ -0,0 +1,239 @@
+import { LRUCache, makeLogger } from '../../lib/index.js'
+
+import { ethers } from 'ethers'
+
+import { funcDTO, txDTO } from './dto.js'
+import { ExecutionApiService } from '../../services/execution-api/service.js'
+
+const ORACLE_FRAME_BLOCKS = 7200
```

Comment:
It could differ on different chains and could be changed on mainnet.
Can we somehow don't use it?

Reply:
Pls, add comment what it is